Squeezer/src/main/java/uk/org/ngo/squeezer/util/ImageWorker.java

```diff
@@ -341,7 +344,17 @@ public static void addDebugSwatch(Canvas canvas, int color) {
      */
     public void addImageCache(ImageCache.ImageCacheParams imageCacheParams) {
         setImageCache(new ImageCache(imageCacheParams));
-        new CacheAsyncTask().execute(MESSAGE_INIT_DISK_CACHE);
+        
+        // AsyncTask instantiates an InternalHandler (which extends Handler but doesn't pass a
```

Comment:
It's hard for me to tell if this actually fixes the problem as it was not easily reproduced. I did not see the problem after I made this change and icons seemed to load fine.

Reply:
I haven't look into the internals of ImageWorker, but the change seems reasonable.